docs/engineering/backend/readme.mdx

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_label: Backend
+---
+
+#
+
+## Backend application setup
+
+The backend requires at a minimum two pieces of software, with an optional third:
+
+1. a FHIR Store, e.g HAPI FHIR
+2. the [FHIR Information Gateway](https://github.com/google/fhir-gateway) with [OpenSRP plugins](https://hub.docker.com/r/onaio/fhir-gateway-plugin/tags)
```

Comment:
The url for FHIR Gateway would be `https://github.com/onaio/fhir-gateway-plugin ` with OpenSRP Plugins.

Reply:
We need to check since we have made the custom repo private.

Reply:
since we're up to date w/upstream we don't need to share that repo